src/pages/RFOX/components/Claim/ClaimRow.tsx

```diff
@@ -75,95 +70,40 @@ export const ClaimRow: FC<ClaimRowProps> = ({
     onClaimClick?.()
   }, [index, onClaimClick, setConfirmedQuote, stakingAmountCryptoBaseUnit, stakingAssetAccountId])
 
-  const parentProps = useMemo(() => {
-    if (onClaimButtonClick) return {}
-
-    return {
-      variant: 'unstyled',
-      as: Button,
-      isDisabled: status !== ClaimStatus.Available,
-      onClick: handleClaimClick,
-      _hover: hoverProps,
-    }
-  }, [onClaimButtonClick, status, handleClaimClick])
-
   const statusText = useMemo(() => {
     if (!isLargerThanMd) return cooldownPeriodHuman
 
-    if (status === ClaimStatus.CoolingDown)
+    if (status === ClaimStatus.NotYetAvailable)
       return translate('RFOX.tooltips.unstakePendingCooldown', { cooldownPeriodHuman })
     return translate('RFOX.tooltips.cooldownComplete', { cooldownPeriodHuman })
   }, [cooldownPeriodHuman, isLargerThanMd, status, translate])
 
-  const actionTranslation = useMemo(() => {
+  const actionText = useMemo(() => {
     if (!stakingAssetSymbol) return
 
-    return [onClaimButtonClick ? actionDescription : 'RFOX.claim', { stakingAssetSymbol }]
-  }, [stakingAssetSymbol, onClaimButtonClick, actionDescription])
+    return translate(onClaimButtonClick ? actionDescription : 'RFOX.claim', { stakingAssetSymbol })
+  }, [stakingAssetSymbol, translate, onClaimButtonClick, actionDescription])
+
+  const tooltipText = useMemo(() => {
```

Comment:
suggestion: `statusText` and `actionText` seems to be the only reason why the `ClaimRow` wrapper exists in the rFOX scope now, we might be able to remove this entire file and handle these texts + the click handler directly in `ClaimSelect` or enrich it with the content in the loop inside ClaimSelect and remove it from the loop to make this wrapper more credible/meaningful

Reply:
Ooh i do like the idea of that, can we do it in a follow up? I see Gomes has some work on this file in draft so it might be wise to minimise diff on it for the short-term?

Reply:
This is a suggestion, feel free to resolve the comment and link it anywhere if you feel like it's interesting enough ;)